GSA/digitalgov.gov
# Update "Connect with us" box on community landing pages

## Summary
Update the "Connect with us" box with community logo, brand colors and survey link to replace old email signup process.

## Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-communities-page/communities/)

<details><summary>Before/After</summary>
<p>

Before
<img width="933" alt="Screenshot 2024-08-27 at 1 44 18 PM" src="https://github.com/user-attachments/assets/4629755c-ccce-4cce-a5a6-d7e4bfc99921">

After
<img width="1027" alt="Screenshot 2024-08-29 at 9 34 30 AM" src="https://github.com/user-attachments/assets/43da997e-194d-424b-9613-8ec42f87998e">

</p>
</details> 

## Solution

Updates the "Connect with us" box to reflect community brand colors and updated copy for the new join button link to the Qualtrics survey signup.

- [x] updated layout for communities page
- [x] updated color for web analytics community
- [x] add survey link

## How To Test

1. Visit any communities page
2. Confirm layout and copy changes 
3. "Join" button links to Qualtrics survey
